domains/summon-the-coder.json

```
@@ -0,0 +1,10 @@
+{   "description": "Site about me",
+    "repo": "https://github.com/banana6boom/about-me",
+    "owner": {
+        "username": "banana6boom",
+        "email": "themadhouse@mail.ru"
+    },
+    "record": {
+        "CNAME": "https://banana6boom.github.io/about-me/"
```

`CNAME` record must contain a domain name only.
```suggestion
        "CNAME": "banana6boom.github.io"
```